templates/web/base/report/_inspect.html

```diff
@@ -43,11 +43,11 @@
           [% cat_prefix = category | lower | replace('[^a-z]', '') %]
           [% cat_prefix = "category_" _ cat_prefix _ "_" %]
           [% IF category == problem.category %]
-            <p data-category="[% category | html %]" data-priorities="[% priorities_by_category.$category %]">
+            <p data-category="[% category | html %]" data-priorities='[% priorities_by_category.$category %]' data-defect-types='[% category_defect_types.$category %]'>
```

Comment:
Fixed this by using `encode_entities`, i.e https://github.com/mysociety/fixmystreet/pull/1698/files#diff-7e531cfc30ebb4348e8295a6a635ebbfR17